http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/BuildHelmChartValues.scala

```diff
@@ -314,10 +314,11 @@ private[leonardo] object BuildHelmChartValues {
                                              leoUrlBase: URL
   ): Values = {
     val relayTargetHost = appType match {
-      case AppType.Cromwell  => s"http://coa-${release.asString}-reverse-proxy-service:8000/"
-      case AppType.Wds       => s"http://wds-${release.asString}-wds-svc:8080"
-      case AppType.HailBatch => "http://batch:8080"
-      case _                 => "uknown"
+      case AppType.Cromwell     => s"http://coa-${release.asString}-reverse-proxy-service:8000/"
+      case AppType.Wds          => s"http://wds-${release.asString}-wds-svc:8080"
+      case AppType.HailBatch    => "http://batch:8080"
+      case AppType.WorkflowsApp => s"http://wfa-${release.asString}-reverse-proxy-service:8000/"
```

Comment:
Can/should we combine the cases with a `|`? Happy to leave it as-is though

Reply:
My thought was (at least internally) it's nice to differentiate coa from wfa (and cra which Saloni will introduce) via prefix, even if they both use reverse proxy service.